### PR TITLE
taydev

### PR DESCRIPTION
**Pull request by taylor-c-williams**

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export default function Header() {
+  const { user, setUser } = useUser();
+  const history = useHistory();
+
+  function logOutButton() {
+    signOutUser();
+    setUser('');
+    history.push('/');
+  }
+  console.log(user);
```
Comment (erichsauer) [inline src/components/Header/Header.jsx:14]: it's okay to leave `console.log` lines in your code, but there should still be an empty line here for readability

File: src/components/PrivateRoute/PrivateRoute.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Redirect, Route } from 'react-router-dom';
+import { useUser } from '../../hooks/useUser';
+
+export default function PrivateRoute({ children, ...routeProps }) {
+  const { user } = useUser();
```
Comment (erichsauer) [inline src/components/PrivateRoute/PrivateRoute.jsx:5]: same here with an empty line - the `return` statement is its own separate stanza in the poem of your code

File: src/context/FormContext.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { createContext, useState } from 'react';
+
+const FormContext = createContext();
```
Comment (erichsauer) [inline src/context/FormContext.jsx:3]: a blank line her would be nice

File: src/context/ProfileContext.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { createContext, useState } from 'react';
+import { getProfile } from '../services/profiles';
+
+const ProfileContext = createContext();
```
Comment (erichsauer) [inline src/context/ProfileContext.jsx:4]: ...as well as here

File: src/context/FormContext.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+const FormProvider = ({ children }) => {
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+
+  return (
+    <FormContext.Provider value={{ email, setEmail, password, setPassword }}>
+      {children}
+    </FormContext.Provider>
+  );
+};
```
Comment (erichsauer) [inline src/context/FormContext.jsx:13]: ...and here

File: src/context/ProfileContext.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        bio,
+        setBio,
+        birthday,
+        setBirthday,
+      }}
+    >
+      {children}
+    </ProfileContext.Provider>
+  );
+};
```
Comment (erichsauer) [inline src/context/ProfileContext.jsx:31]: ...and here

**Review by erichsauer — APPROVED** [review]
I love love love your app planning image ❤️

**Comment by taylor-c-williams** [comment]
I can’t reply to this can I? Thanks! I’m about to start spiffing up my gh
so I’m trying to have relevant readmes now

**Pull request merged**